### ISSUE TITLE
Improve UI prompt buffer bookmark facilities

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1522]
After `bookmark-current-url`, the prompt buffer receives tags from the user:

![image](https://user-images.githubusercontent.com/7698207/122494206-4fd51580-cfbf-11eb-9d6e-6289b51dd16b.png)

I think the **default** should be to have **all** tags typed by the user implicitly selected:
![image](https://user-images.githubusercontent.com/7698207/122495392-5617c180-cfc0-11eb-9c9b-e9be230ec0be.png)

If he/she wants to remove something, the person can do `M-u`. As it is now, the user needs to press `M-a` to select what he/she has just typed. Choosing the most reasonable default seems to make a lot of difference for productivity. I might have missed something about the current design, though.

**Comment by Ambrevar** [comment]
True, the user experience of tag selection can be a bit confusing.

The thing is, we don't automatically select new tags because the user input might be used to narrow down existing tags.

Say, `foobar` is an existing tag.
Now if I want to bookmark a page with tag `foobar`, I can just type `foo`, then select `foobar` in the narrowed-down list.  On `return`, I don't want to tag with `foo` however!  So we can't automatically select.

If you have suggestion about how to reconcile those two angles, you are more than welcome :)

**Comment by jmercouris** [comment]
Pierre is right, I don't think this can be reconciled.
